### ISSUE TITLE
projects/core/howto/systemd.rst should be properly formatted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6979]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)| @wallrj reported|
|-|-|
|Trac ID|trac#6979|
|Type|task|
|Created|2014-02-19 17:29:07Z|
|Branch|https://github.com/twisted/twisted/tree/systemd-howto-formatting-6979|

Similar to previous work by hawkowl (eg [#6943](https://github.com/twisted/twisted/issues/6943)), this is about updating the rST sources to be correctly formatted.

But also to fix various hyperlinks to systemd man pages which seem to have been messed up by lor2sphinx.

 * http://twisted.readthedocs.org/en/latest/projects/core/howto/systemd.html

<details><summary>Searchable metadata</summary>

```
trac-id__6979 6979
type__task task
reporter__rwall rwall
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_systemd_howto_formatting_6979 branches/systemd-howto-formatting-6979
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1392830947000000 1392830947000000
changetime__1392896812000000 1392896812000000
version__None None
owner__rwall rwall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_1"></a>|@wallrj commented|
|-|-|

(In [41716]) Branching to 'systemd-howto-formatting-6979'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_2"></a>|@wallrj removed owner|
|-|-|

Ready for review in log:branches/systemd-howto-formatting-6979 
 * lore2sphinx had broken all the configuration file listings
 * broken many of the links
 * Lost whole sentences while failing to deal with link text that contained nested tags
 * Added extra space between `````code``}} blocks and the following full stop (or other punctuation)
 * Failed to handle strike-through text.

I think I've fixes all those things and also converted to one-sentence-per-line and trimmed down the whitespace.

Build results:
 * https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/systemd-howto-formatting-697```{``code``}} blocks and the following full stop (or other punctuation)
 * Failed to handle strike-through text.

I think I've fixes all those things and also converted to one-sentence-per-line and trimmed down the whitespace.

Build results:
 * https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/systemd-howto-formatting-6979

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @wallrj|
|-|-|

LGTM please land.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set status to `closed`|
|-|-|

(In [41736]) Merge systemd-howto-formatting-6979

Author: rwall
Reviewer: glyph
Fixes: [#6979](https://github.com/twisted/twisted/issues/6979)

Fix broken links and formatting in the ReST version of the systemd howto
document.
